### PR TITLE
add store to centralize call to API

### PR DESCRIPTION
**Pull request by etnbrd**
This PR reintroduces the store in a more centralized fashion now that we have an SPA.

### CODE REVIEW

File: dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export type OpenAPIConfig = {
 };
 
 export const OpenAPI: OpenAPIConfig = {
-    BASE: process.env.NODE_ENV === 'production' ? '/' : 'http://127.0.0.1:8000',
+    BASE: process.env.NODE_ENV === 'production' ? '' : 'http://127.0.0.1:8000',
```
Comment (brouberol) [inline dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts:23]: Due to the fact that you removed the Makefile rule for this file, next time it's generated, it'll be reverted back to `BASE: ''` which is alright, now that we have an SPA, I think.

```suggestion
    BASE: '',
```

**Comment by brouberol** [comment]
Relies on https://github.com/brouberol/5esheets/pull/148

File: dnd5esheets/front/src/components/CharacterSheet.tsx [hunk 2]
```diff
@@ -182,22 +182,26 @@ export default function CharacterSheet({
...
+                    ] as const
+                  }
                 >
                   {(attribute) => (
                     <ScoreBox
                       label={t(`${attribute}_abbr`)}
-                      score={character.data.scores[attribute]}
-                      modifier={character.data.scores[`${attribute}_mod`]}
+                      score={character.data.abilities[attribute].score}
+                      modifier={character.data.abilities[attribute].modifier}
```
Comment (brouberol) [inline dnd5esheets/front/src/components/CharacterSheet.tsx:200]: **Note**: This relies on https://github.com/brouberol/5esheets/pull/148

**Review by brouberol — COMMENTED** [review]
Nicely done! I only have one nit about the `BASE` openAPI client option.

File: dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts (hunk 1 above)
Comment (etnbrd) [reply dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts:23]: Ah! I missed that. Let's merge it anyway, and we'll find a fix later.

Comment (brouberol) [reply dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts:23]: I think that you can just accept the suggestion and we should be alright

**Review by brouberol — APPROVED** [review]
✨

File: dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts (hunk 1 above)
Comment (etnbrd) [reply dnd5esheets/front/src/5esheets-client/core/OpenAPI.ts]: fixed by: https://github.com/brouberol/5esheets/pull/158/commits/4d69abe279fb2f5dfbcbd38c0a0738a3657d691c

> I think that you can just accept the suggestion and we should be alright

We still need the conditional somewhere.

**Pull request merged**